### PR TITLE
Forward Input Events to GVRViews

### PR DESCRIPTION
**Pull request by rahul27**
The sample forwards key events generated by Android
to a GVRView.

Make use of the Gaze Controller to synthesize motion events
for GVRViews - use the cursor to click on buttons.

The sample also prints the headset hardware button presses
like back, volume up and down.

The sample only works with a VR headset.

### CODE REVIEW

**Comment by rahul27** [comment]
This sample depends on pull request https://github.com/Samsung/GearVRf/pull/414.

**Comment by thomasflynn** [comment]
nice.  +1 Verified.
question:  if i had a ListView, could i swipe up/down to scroll?

**Comment by rahul27** [comment]
@thomasflynn Yup. I've added a ListView to the sample.

**Comment by thomasflynn** [comment]
nice!  +1 Verified

File: events-sample/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="org.gearvrf.input.events"
+    android:versionCode="1"
+    android:versionName="1.0" >
+
+    <uses-permission android:name="android.permission.INTERNET"/>
```
Comment (liaxim) [inline events-sample/AndroidManifest.xml]: @rahul27 READ_EXTERNAL_STORAGE is missing; per https://developer.oculus.com/documentation/publish/latest/concepts/publish-mobile-manifest/

Comment (rahul27) [inline events-sample/AndroidManifest.xml]: Ah, I based this out of the RenderableView project. In any case I've uploaded a new pull request to fix both projects:

https://github.com/gearvrf/GearVRf-Demos/pull/40

**Pull request merged**